### ISSUE TITLE
stop release-please from blowing away manual edits

### ISSUE THREAD

**Issue by codyoss** [googleapis/release-please#877]
**Is your feature request related to a problem? Please describe.**

Sometimes a maintainer may need to edit a release PR in order to provide extra context, deprecation/migration warnings, etc to the release notes. Currently if a person does this and another commits comes in after the changes were made and before the PR was merged those manual edits get blown away.

 **Describe the solution you'd like**

It would be nice is release-please would honor a Github label and no longer push to the branch.

 **Describe alternatives you've considered**

Another approach would be to have a command comment to due something similar. Example: `/stopUpdates`

**Extra context**

Either way in this case I think release please may need to keep track of what commit it should tag at.

**Comment by voxpelli** [comment]
I agree, this was the worry I expressed here https://github.com/eslint/eslint/issues/14231#issuecomment-862574734

I think it would be nice to have a section of the file which is always left untouched and also have a way to completely lock the files from changes (and maybe eg. having comments on the PR instead whenever an update were to be applied but instead is ignored)

**Comment by luxaritas** [comment]
A couple related things that would help this use-case:
- Changes to CHANGELOG files comitted to the release branch being reflected in the PR body and eventual github release
- Not sure how viable it would be, but an in addition to a mode where you prevent updates completely, a mode where you only add new line items and don't "regenerate" previous items or touch any other areas of the changelog (eg, a header or footer)

**Comment by shinebayar-g** [comment]
Commit bodies are also ignored https://github.com/googleapis/release-please/issues/1957#issuecomment-1559592931
If bodies weren't ignored we could at least use `BEGIN_COMMIT_OVERRIDE, END_COMMIT_OVERRIDE` option to edit the PR body and generate decent changelog we want.

**Comment by awoimbee** [comment]
Related issues: https://github.com/googleapis/release-please/issues/1371, https://github.com/googleapis/release-please/issues/1713,  https://github.com/googleapis/release-please/issues/2042, https://github.com/googleapis/release-please/issues/2458

Looks to me this could also supplant `release-as` and close bug tickets related to that feature: #2704, #2349, #2231, #1905
